hip/components/uninitialized_array.hip.hpp

```diff
@@ -50,4 +50,4 @@ namespace hip {
 }  // namespace gko
 
 
-#endif  // GKO_HIP_COMPONENTS_UNINITIALIZED_ARRAY_HIP_HPP_
+#endif  // GKO_HIP_BASE_COMPONENTS_ARRAY_HIP_HPP_
```

Comment:
A search replace error ?
```suggestion
#endif  // GKO_HIP_COMPONENTS_UNINITIALIZED_ARRAY_HIP_HPP_
```